_src/static/sass/utilities/_variables.scss

```diff
@@ -233,6 +230,17 @@ $font-weights: (
     x-bold:         800
 );
 
+// typography: letter-spacing
+$letter-spacing: (
+    normal:         0,
+    tightest:       -3,
+    tighter:        -2,
+    tight:          -.095rem,
+    loose:          -.075rem,
+    looser:         -.025rem,
+    loosest:        3
+);
+
```

Comment:
Also, please loose the ``rem`` units from this scale, I'd prefer them to be added in the ``letter-spacing`` function.